modules/candidate_parameters/ajax/getData.php

```diff
@@ -490,3 +493,29 @@ function getConsentStatusHistory($pscid)
     }
     return $formattedHistory;
 }
+
+/**
+ * Handles the fetching of candidate's date of death.
+ *
+ * @throws DatabaseException
+ *
+ * @return array
+ */
+function getDODFields(): array
+{
+    $candID = $_GET['candID'];
```

Comment:
```suggestion
    $candID = new CandID($_GET['candID']);
```

Just to make sure that the input is valid. You may get a security issue here if you later return user-supplied data to the front-end. Passing it through this class will make sure that the user supplied a valid CandID first.

You'll probably want to wrap this in a try-catch with a \DomainException and return an error if encountered.